modules/imgproc/perf/perf_goodFeaturesToTrack.cpp

```diff
@@ -31,14 +31,16 @@ PERF_TEST_P(Image_MaxCorners_QualityLevel_MinDistance_BlockSize_gradientSize_Use
         FAIL() << "Unable to load source image" << filename;
 
     std::vector<Point2f> corners;
+    std::vector<float> corners_values;
 
     double minDistance = 1;
-    TEST_CYCLE() goodFeaturesToTrack(image, corners, maxCorners, qualityLevel, minDistance, noArray(), blockSize, gradientSize, useHarrisDetector);
+    TEST_CYCLE() goodFeaturesToTrack(image, corners, maxCorners, qualityLevel, minDistance, noArray(), blockSize, gradientSize, useHarrisDetector, 0.04, corners_values);
```

Comment:
This change drops testing of old function.

You should create new one instead.